src/presets/node.ts

```diff
@@ -8,4 +9,5 @@ import { XMLHttpRequestInterceptor } from '../interceptors/XMLHttpRequest'
 export default [
   new ClientRequestInterceptor(),
   new XMLHttpRequestInterceptor(),
+  new FetchInterceptor(),
```

Comment:
curious what the status is here, since I think this would be something we'd want to support cloudflare like environments?